ocaml/xapi/certificates.ml

```diff
@@ -239,19 +247,9 @@ end = struct
           D.error
```

Comment:
Should this be a warning now that we don't raise anymore?

Reply:
rethinking about it... yes, there are too many scary errors around the logs, let's mark it as warning